frontend/src/components/Toast/Toast.tsx

```diff
@@ -44,13 +52,38 @@ export const Toast = ({
     variant: status,
   })
 
+  const StatusIcon = useMemo(
+    () => (status === 'success' ? BxsCheckCircle : BxsErrorCircle),
+    [status],
+  )
+
+  const mdComponents = useMdComponents(styles)
+
+  const descriptionComponent = useMemo(() => {
+    // ReactMarkdown requires children to be of string type. If description is
+    // not a string, return description as is.
+    if (!description || !useMarkdown || typeof description !== 'string')
+      return description
+    return (
+      <ReactMarkdown components={mdComponents}>{description}</ReactMarkdown>
+    )
```

Comment:
wondering if we should extract this into its own component?

Reply:
I think its fine, since mdComponents has the possibility to be unique with its given styles